webpack.config.js

```diff
@@ -1,25 +1,22 @@
 'use strict';
 
-const webpack = require('webpack');
+//const webpack = require('webpack');
```

Comment:
do you want this removed or just commented out

Reply:
Good call.  I'll delete it.